### ISSUE TITLE
How to divide data

### ISSUE THREAD

**Issue by Seilmast** [SFI-Visual-Intelligence/Collaborative-Coding-Exam#12]
We need to make sure we have a train and validation set for our code. There are two methods which I know off to solve this:

1. Use `torch.utils.data.random_split`. This means we don't have to manually split the data while creating the dataset handlers
2. Manually handle the dataset and return a separate train and validation data loader class. 

Let me know which approach you prefer.

**Comment by salomaestro** [comment]
I prefer option 1.

**Comment by sot176** [comment]
Option 1 sounds good.

**Comment by hzavadil98** [comment]
Option 1 definitely sounds better, with fixing a seed somewhere so that we are consistent at least for start..

**Comment by Seilmast** [comment]
We're going with option 1 then. Closing this issue
